src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -38,111 +72,49 @@ namespace AppInstaller::CLI::Workflow
+        ShowSingleLineField(info, Resource::String::ShowLabelPrivacyUrl, manifest.CurrentLocalization.Get<Manifest::Localization::PrivacyUrl>());
+        ShowSingleLineField(info, Resource::String::ShowLabelCopyright, manifest.CurrentLocalization.Get<Manifest::Localization::Copyright>());
+        ShowSingleLineField(info, Resource::String::ShowLabelCopyrightUrl, manifest.CurrentLocalization.Get<Manifest::Localization::CopyrightUrl>());
+        ShowMultiLineField(info,  Resource::String::ShowLabelReleaseNotes, manifest.CurrentLocalization.Get<Manifest::Localization::ReleaseNotes>());
+        ShowSingleLineField(info, Resource::String::ShowLabelReleaseNotesUrl, manifest.CurrentLocalization.Get<Manifest::Localization::ReleaseNotesUrl>());
+        ShowSingleLineField(info, Resource::String::ShowLabelPurchaseUrl, manifest.CurrentLocalization.Get<Manifest::Localization::PurchaseUrl>());
+        ShowMultiLineField(info, Resource::String::ShowLabelInstallationNotes, manifest.CurrentLocalization.Get<Manifest::Localization::InstallationNotes>());
         const auto& documentations = manifest.CurrentLocalization.Get<Manifest::Localization::Documentations>();
         if (!documentations.empty())
         {
-            context.Reporter.Info() << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelDocumentation << std::endl;
+            ShowSingleLineField(info, Resource::String::ShowLabelDocumentation, " "_liv);
             for (const auto& documentation : documentations)
             {
                 if (!documentation.DocumentUrl.empty())
                 {
                     info << "  "_liv;
                     if (!documentation.DocumentLabel.empty())
                     {
-                        info << Execution::ManifestInfoEmphasis << documentation.DocumentLabel << ": "_liv;
+                       info << Execution::ManifestInfoEmphasis << documentation.DocumentLabel << ": "_liv;
                     }
-
                     info << documentation.DocumentUrl << std::endl;
                 }
             }
         }
-        const auto& tags = manifest.CurrentLocalization.Get<Manifest::Localization::Tags>();
-        if (!tags.empty())
-        {
-            context.Reporter.Info() << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelTags << std::endl;
-            for (const auto& tag : tags)
-            {
-                info << "  "_liv << tag << std::endl;
-            }
-        }
+        ShowMultiValueField(info, Resource::String::ShowLabelTags, manifest.CurrentLocalization.Get<Manifest::Localization::Tags>());
         const auto& agreements = manifest.CurrentLocalization.Get<Manifest::Localization::Agreements>();
         if (!agreements.empty())
         {
-            context.Reporter.Info() << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelAgreements << std::endl;
+            ShowSingleLineField(info, Resource::String::ShowLabelAgreements, " "_liv);

```

Comment:
Not a fan of reusing the function for something that is not semantically correct.